packages/block-library/src/image/edit.native.js

```diff
@@ -85,6 +87,10 @@ class ImageEdit extends React.Component {
 	}
 
 	componentWillUnmount() {
+		// this action will only exist if the user pressed the trash button on the block holder
+		if ( hasAction( 'blocks.onRemoveBlockCheckUpload' ) && this.state.isUploadInProgress ) {
```

Comment:
Maybe it would be good to put the action name in some constant so that we can reuse it?

Reply:
Following with comment here https://github.com/wordpress-mobile/gutenberg-mobile/pull/736#discussion_r265971739

Another thing I just noticed is that the constant would need to be defined in gutenberg-mobile and be imported in Gutenberg native file (as those are the 2 places where it's used, for adding in the Gutenberg-mobile level and removing at the Gutenberg component level) - I don't think there's much gain as there's 2 places only, been thinking of doing it through props as well, but it makes the code a small bit harder to follow. Would it be ok to leave it as is ? wdyt?